test/unit/shimming.test.js

```diff
@@ -31,14 +31,25 @@ describe('shimming', function () {
   });
 
   afterEach(function () {
-    [
-      connection.Model.prototype.findByPrimary,
-      connection.Model.prototype.findById,
+    const wrapped = [
       connection.Association.BelongsTo.prototype.get,
       connection.Association.HasOne.prototype.get,
       connection.Association.HasMany.prototype.get,
       connection.Association.BelongsToMany.prototype.get
-    ].forEach(i => i.__unwrap && i.__unwrap());
+    ];
+
+    if ( /^4/.test(Sequelize.version) ) {
+      wrapped.push(
+        connection.Model.findById
```

Comment:
Right, I'm still implementing my forks in my project and testing things. I'll get back to these PRs soon.